### ISSUE TITLE
Export notebook as script not showing / Help > Classic Notebook not working

### ISSUE THREAD

**Issue by Apotygma** [jupyterlab/jupyterlab#9603]
## Description

Hi all,

I recently decided to get rid of Anaconda and run a clean version of python. In that move, I moved from Jupyter notebook to Jupyter Lab (version 3.0 just came out).

I ran in the issue that I could not export my notebooks as scripts. It does not even show in the options
![2021-01-12_15-01-05](https://user-images.githubusercontent.com/76772177/104324057-35b27b00-54e7-11eb-98c9-42dc349ebd99.jpg)

## Solutions tried

From following error #8028, I had two options that gave additional errrors
- Reinstalling nbconvert does not change anything
- Opening Classic Notebook

### Reinstalling Nbconvert

Issues that I see:

- I don't have the options
- Even through command line, I don't have the options

<pre>
--to=<Unicode>
    The export format to be used, either one of the built-in formats ['custom',
    'latex', 'latex_with_lenvs', 'notebook', 'pdf', 'selectLanguage'] or a
    dotted object name that represents the import path for an `Exporter` class
    Default: ''
    Equivalent to: [--NbConvertApp.export_format]
</pre>

Issue seems to be that nbconvert it's installed in two locations
![2021-01-12_15-19-30](https://user-images.githubusercontent.com/76772177/104326239-cab67380-54e9-11eb-96d9-4fc8f8ef13d2.jpg)

### Opening Classic Notebook

This option does not work
![2021-01-12_15-01-16](https://user-images.githubusercontent.com/76772177/104326796-6a740180-54ea-11eb-81dc-a92922886f15.jpg)

I can however open in through command line "Jupyter Notebook"
![2021-01-12_15-26-09](https://user-images.githubusercontent.com/76772177/104326975-9abba000-54ea-11eb-961b-9467e6c74900.jpg)

What can i do to solve all this ?
What did I do wrong during uninstallation of anaconda and reinstall?

Regards

Mathias

**Comment by jasongrout** [comment]
It sounds like there are installation issues here (duplicate installations, etc.). If I were you, I'd probably uninstall (and make sure *everything* was uninstalled), then reinstall.

I just verified that a clean install of JLab 3 in a new environment (something equivalent to the below) has exports from nbconvert and the "Launch Classic" menu item working.

```
conda create -n test -c conda-forge jupyterlab=3 python=3 
```

**Comment by Apotygma** [comment]
Hey

Thanks for the feedback. Shamefully, unsinstalling everything, deleting all files manually (.jupyter / .python / .anaconda and anything else I could find related to python) did not solve my issue 😭 

I did find what I did wrong during anaconda uninstall, apparently they recommend to run anaconda-clean to delete everything. That's why I tried deleting everything manually.

**NB Convert not working**
nbconvert is still installing in two different locations
![2021-01-14_10-19-26](https://user-images.githubusercontent.com/76772177/104574611-8694b180-5656-11eb-966c-56f6dd3b0917.jpg)

Simply moving the templates from one to the other solved the issue, but not ideal obviously.

**Jupter Notebook not working**
Same issue remains, I can open Lab fine through CMD, I can open notebook fine through CMD, but I can't switch within lab to notebook.


Could this issue be related to an error message I get while install (not my error message, but I got the same one)?
`Defaulting to user installation because normal site-packages is not writeable Requirement already satisfied: pip in /home/stu16291/.local/lib/python3.6/site-packages (20.3.1)`

Can I simply move some files from directory to solve this? :smile:

Thanks for the feedback anyway

**Comment by jasongrout** [comment]
My guess is that the notebook that is installed is not in the same environment as the lab that is installed. To switch from lab to notebook, all it does is change the url from `/lab` to `/tree`. Yes, it does seem like this could be a symptom of the error that you saw - it is installing lab in a completely separate place from the notebook.

>  `Defaulting to user installation because normal site-packages is not writeable Requirement already satisfied: pip in /home/stu16291/.local/lib/python3.6/site-packages (20.3.1)`

This sounds like the problem with the nbconvert - it sounds like you have two nbconvert installations, and somehow that is getting things messed up.

Perhaps you should find out why that normal installation directory is not writeable and fix that.

**Comment by Apotygma** [comment]
Hey

Thanks for the feedback, I've managed to make it work.

Root causes (likely):
- I have a corporate laptop, meaning that I have administrator rights but my profile is not admin by default. When installing exe's there is a prompt to validate, there is not in CMD.
- I didn't run CMD as administator
- I switched from a user python install to a global python install. This meant that the site-packages are on my C: drive in Program Files now, no long in "AppData". There, again, there are issues with Admin rights. My normal profile does not have rights to edit the folders located there. I did fix this through Options > Security and then giving read/write files to all profiles.
- I now run the install also through "Python -m pip" instead of just using pip

Now it's all working as expected.

Thanks again
